<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QSpinBox" name="spinBox">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="ToolBarLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="OkayPush">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="PlotWidget" name="View"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionaddPoints"/>
   <addaction name="actionouter"/>
   <addaction name="actioninner"/>
   <addaction name="actioninsertionpoints"/>
   <addaction name="actionTranslate"/>
   <addaction name="actionscale"/>
   <addaction name="actionactionXscale"/>
   <addaction name="actionYscale"/>
   <addaction name="actioncircle"/>
   <addaction name="actionelipse"/>
   <addaction name="actiondeleteShape"/>
   <addaction name="actionDelPoint"/>
   <addaction name="actionZoom"/>
   <addaction name="actionedit"/>
   <addaction name="actionRoundPointer"/>
   <addaction name="actionregions"/>
   <addaction name="actionsegments"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionOpen_Dicom"/>
  </widget>
  <action name="actionOpen_Dicom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>file_open_FILL0_wght700_GRAD0_opsz48.png</normaloff>file_open_FILL0_wght700_GRAD0_opsz48.png</iconset>
   </property>
   <property name="text">
    <string>Open Dicom</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>search_FILL0_wght700_GRAD0_opsz48.png</normaloff>search_FILL0_wght700_GRAD0_opsz48.png</iconset>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
  </action>
  <action name="actionaddPoints">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>shape_line_FILL0_wght700_GRAD0_opsz48.png</normaloff>shape_line_FILL0_wght700_GRAD0_opsz48.png</iconset>
   </property>
   <property name="text">
    <string>addPoints</string>
   </property>
  </action>
  <action name="actiondeleteShape">
   <property name="icon">
    <iconset>
     <normaloff>delete_FILL0_wght700_GRAD0_opsz48.png</normaloff>delete_FILL0_wght700_GRAD0_opsz48.png</iconset>
   </property>
   <property name="text">
    <string>deleteShape</string>
   </property>
  </action>
  <action name="actionDelPoint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>block_FILL0_wght700_GRAD0_opsz48.png</normaloff>block_FILL0_wght700_GRAD0_opsz48.png</iconset>
   </property>
   <property name="text">
    <string>DelPoint</string>
   </property>
  </action>
  <action name="actionTranslate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>open_with_FILL0_wght700_GRAD0_opsz48.png</normaloff>open_with_FILL0_wght700_GRAD0_opsz48.png</iconset>
   </property>
   <property name="text">
    <string>Translate</string>
   </property>
  </action>
  <action name="actionincrease">
   <property name="text">
    <string>increase</string>
   </property>
  </action>
  <action name="actiondecrease">
   <property name="text">
    <string>decrease</string>
   </property>
  </action>
  <action name="actionscale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>aspect_ratio_FILL0_wght700_GRAD0_opsz48.png</normaloff>aspect_ratio_FILL0_wght700_GRAD0_opsz48.png</iconset>
   </property>
   <property name="text">
    <string>scale</string>
   </property>
  </action>
  <action name="actionYscale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resizenorthsouth_100184.png</normaloff>resizenorthsouth_100184.png</iconset>
   </property>
   <property name="text">
    <string>Yscale</string>
   </property>
  </action>
  <action name="actionactionXscale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>resizeeastwest_100180.png</normaloff>resizeeastwest_100180.png</iconset>
   </property>
   <property name="text">
    <string>Xscale</string>
   </property>
  </action>
  <action name="actioncircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>circle_FILL0_wght400_GRAD0_opsz48.png</normaloff>circle_FILL0_wght400_GRAD0_opsz48.png</iconset>
   </property>
   <property name="text">
    <string>circle</string>
   </property>
  </action>
  <action name="actionelipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>ellipse_adobe_illustrator_tool_circle_icon_189052.png</normaloff>ellipse_adobe_illustrator_tool_circle_icon_189052.png</iconset>
   </property>
   <property name="text">
    <string>elipse</string>
   </property>
  </action>
  <action name="actionedit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>draw_FILL0_wght400_GRAD0_opsz48.png</normaloff>draw_FILL0_wght400_GRAD0_opsz48.png</iconset>
   </property>
   <property name="text">
    <string>edit</string>
   </property>
  </action>
  <action name="actionRoundPointer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>radio_button_checked_FILL0_wght500_GRAD0_opsz48.png</normaloff>radio_button_checked_FILL0_wght500_GRAD0_opsz48.png</iconset>
   </property>
   <property name="text">
    <string>RoundPointer</string>
   </property>
  </action>
  <action name="actionouter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>outter</string>
   </property>
  </action>
  <action name="actioninner">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>inner</string>
   </property>
  </action>
  <action name="actionregions">
   <property name="icon">
    <iconset>
     <normaloff>donut_small_FILL1_wght500_GRAD200_opsz48.png</normaloff>donut_small_FILL1_wght500_GRAD200_opsz48.png</iconset>
   </property>
   <property name="text">
    <string>regions</string>
   </property>
  </action>
  <action name="actioninsertionpoints">
   <property name="text">
    <string>insertionpoints</string>
   </property>
  </action>
  <action name="actionsegments">
   <property name="text">
    <string>segments</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
